src/briefcase/windows_icon.py

```diff
@@ -0,0 +1,92 @@
+from ctypes import *
+from win32ctypes.pywin32 import win32api
```

Comment:
At column 1: (F403) 'from ctypes import *' used; unable to detect undefined names